client/asset/dcr/dcr.go

```diff
@@ -1146,12 +1178,19 @@ func (dcr *ExchangeWallet) Lock() error {
 
 // PayFee sends the dex registration fee. Transaction fees are in addition to
 // the registration fee, and the fee rate is taken from the DEX configuration.
-func (dcr *ExchangeWallet) PayFee(address string, regFee uint64, nfo *dex.Asset) (asset.Coin, error) {
+func (dcr *ExchangeWallet) PayFee(address string, regFee uint64) (asset.Coin, error) {
 	addr, err := dcrutil.DecodeAddress(address, chainParams)
 	if err != nil {
 		return nil, err
 	}
-	msgTx, sent, err := dcr.sendRegFee(addr, regFee, nfo)
+	feeRate, err := dcr.FeeRate()
+	if err != nil {
+		feeRate = dcr.fallbackFeeRate
+		dcr.log.Errorf("PayFee: Unable to get optimal fee rate, using fallback %d: %v",
+			dcr.fallbackFeeRate, err)
+	}
+	// TODO: why not SendToAddress like in btc ExchangeWallet?
```

Comment:
If I'm remembering correctly, it's because dcrwallet randomizes the change output index, so creating the coin would require a bit of extra work.